src/Hazelcast.Net.Tests/Resources/Options/HazelcastFailoverOptions.json

```diff
@@ -0,0 +1,299 @@
+{
+  
+  "hazelcast-failover": {
+    "tryCount": 42,
+    "clusters": [
+      {
+        // name of the client
+        "clientName": "client",
+
+        // name of the cluster
+        "clusterName": "cluster",
+
+        // how long to pause before looking for clients again, when trying to
+        // set the client that handles cluster view events, and no client is
+        // available
+        "waitForClientMilliseconds": 1001,
```

Comment:
Is this a valid configuration option? I couldn't see it anywhere apart from this

Reply:
I think we have discussed it before. It is used on retry strategy to expire the retry.

Reply:
Can you double check? I looked at the code in the master, but the only place we mention this configuration option is a file named `HazelcastOptions.json`, which feels like it is a leftover